### ISSUE TITLE
This patch won't apply to RC 9.9.0

### ISSUE THREAD

**Issue by pbchase** [ctsit/change_from_address_in_dashboard_reminder#2]
This patch won't apply to RC 9.9.0. It need to work with 9.9.0 while still working with 9.3.5. 

See https://github.com/ctsit/shibboleth_and_table_auth/blob/master/apply_patch.sh got good code to steal.

**Comment by ChemiKyle** [comment]
Specific useful lines 15-27:

```bash
PATCH_VERSIONS=$(mktemp)
$(ls *.patch | grep -v "sql" | sed "s/redcap-//; s/\.patch//;" > $PATCH_VERSIONS)
if [[ $(cat $PATCH_VERSIONS | grep "$REDCAP_VERSION") ]]; then
    REDCAP_PATCH_VERSION=$REDCAP_VERSION
else
    echo "${REDCAP_VERSION}" >> "$PATCH_VERSIONS"
    if [[ "$(cat $PATCH_VERSIONS | sort -V | head -n 1)" != "$MIN_REDCAP_PATCH_VERSION" ]]; then
        echo "${REDCAP_VERSION} is below the minimum supported version, ${MIN_REDCAP_PATCH_VERSION} and will not be patched. Aborting process."
        rm $PATCH_VERSIONS
        exit
    fi
    REDCAP_PATCH_VERSION=$(cat $PATCH_VERSIONS | sort -V | grep "$REDCAP_VERSION" -B 1 | head -n1)
fi
```

**Comment by ChemiKyle** [comment]
Interesting side note: given a set of semantic version strings and the string "patch", `sort -V` considers "patch" to be the highest version, unbound by the fetters of majors, minors, and mere mortals it soars to the top like a phoenix to annoy me.

**Comment by pbchase** [comment]
closed by PR #3
